README.md

```diff
@@ -12,6 +12,11 @@ Automate fixing deprecated Drupal code.
 
 ### Release notes
 
+* The 0.18.0 and higher releases of drupal-rector will include Rector 0.18+. The upgrade path should be as simple as re-copying the configuration file. `cp vendor/palantirnet/drupal-rector/rector.php`.
+
+> [!WARNING]
+> Drupal rector does not support Drupal 10.0 specifically because this will break support for backwards compatible fixes for which the helpers were introduced in Drupal 10.1. Older and newer versions of Drupal are supported.
```

Comment:
This confused me, what specifically was changed here? Why can't we just fix them as non-backward compatible like previous version?

Reply:
Yeah we can, but question is if we want that split. Was also thinking about update not. But I'll just run 10.1 and adjust changes to the info.yml accordingly.

 So I will remove this part